package.json

```diff
@@ -3,19 +3,23 @@
   "version": "0.1.5",
   "description": "TypeScript client for Weaviate",
   "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "types": "types/index.d.ts",
   "engines": {
     "node": ">=16.0.0"
   },
   "files": [
-    "dist/index.js",
-    "dist/index.js.map",
-    "dist/**/*.d.ts"
+    "/dist/index.js",
+    "/dist/index.js.map",
+    "/types/**/*.d.ts"
   ],
   "scripts": {
-    "test": " tsc -noEmit -p tsconfig-test.json && jest --useStderr --runInBand --detectOpenHandles --forceExit",
-    "build": "tsc --emitDeclarationOnly && ./build.js",
-    "prepack": "npm run build"
+    "test": "tsc -noEmit -p tsconfig-test.json && jest --useStderr --runInBand --detectOpenHandles --forceExit",
+    "build": "npm run lint && tsc && ./build.js",
```

Comment:
Why was the `--emitDeclarationOnly` removed?

Reply:
No preference - I think it was just because merging changes from the two repos.